### DISCUSSION TITLE
Under review for a month

### DISCUSSION THREAD

**Discussion by daniel-soli** [WalletConnect/walletconnect-monorepo#4418] [Developer Support]
Hi, I own layerloot.io and are using web3modal (formerly used rainbowkit). I have submitted my app for publish in the explorer, but it has been under review for about a month, if not longer.
I have tried to make changes to it so that I could submit a updated version, but that hasn't helped. 

Also, when doing testing I can see in the cookies that the dApp name are set to LooperLands which is not my app and I am not using the same project id or anything as they do (how would I even know their project id).

Any help in getting my project reviewed and accepted would be nice :)

**Comment by glitch-txs** [comment]
hi, what's your project ID?

**Comment by Cali93** [comment]
Hey @daniel-soli,

Explorer listings for dapps is being deprecated. We don't review and approve dapps anymore.
We will update the Cloud dashboard shortly to reflect that.
